.travis.yml

```diff
@@ -0,0 +1,14 @@
+dist: trusty
+sudo: false
+git:
+  depth: 5
+ 
+language: python
+python: 3
+
+cache: pip
+install:
+  - mkvirtualenv cdtn-scripts
+  - pip3 install -r requirements.txt
+  
+script: pipenv run pylint --output-format=colorized search
```

Comment:
<img src=https://user-images.githubusercontent.com/730511/51255528-bc3d2c00-19a3-11e9-97f8-07e303e79f91.jpg width=693>

Reply:
I don't think we can both use pipenv and mkvirtualenv, so I'll try and fix this. Can you plug travisci to this repository so I can see how it goes @douglasduteil ? thanks